src/index.js

```diff
@@ -1,5 +1,10 @@
-// import _ from 'lodash';
 import './style.css';
-import Todo from './module/createList.js';
 
-Todo();
+import TaskList from './module/TaskList.js';
+
+const planned = document.querySelector('#tasks_lists');
+const todoInput = document.querySelector('#todo_input');
+const todoBtn = document.querySelector('#todo_btn');
+
+const taskList = new TaskList(planned, todoInput, todoBtn);
+taskList.init();
```

Comment:
- [ ] It would be better to add the edit task so that the user can edit a task if he/she makes a typo and also complete the requirements 
<img width="502" alt="Screenshot 2023-05-10 at 22 59 41" src="https://github.com/zbahati/Todo-list/assets/61790587/97681af9-5f0f-490a-9afd-bec4925d3fd4">

Reply:
I add edit functionality so that user can edit whenever he/she want do edit